src/vm/appdomain.cpp

```diff
@@ -3877,7 +3877,6 @@ AppDomain::AppDomain()
     m_dwFlags = 0;
     m_pDefaultContext = NULL;
 #ifdef FEATURE_COMINTEROP
-    m_pComCallWrapperCache = NULL;
     m_pRCWCache = NULL;
```

Comment:
Are the RCW caches fine wrt. unloading?

Reply:
They (or I should rather say it as there is only one in one app domain) don't cause any issues while unloading. It seems that the only reason why they were per AppDomain was the isolation (based on the doc I was able to find in BOTR) and they seem to be cleaned on a regular basis.

Reply:
E.g. RCW has `InterfaceEntry m_aInterfaceEntries[INTERFACE_ENTRY_CACHE_SIZE];` in it. This array contains MethodTable pointers. What guarantees that none of these pointers point to MethodTable that got unloaded?

Reply:
Let me look into that